src/models.py

```diff
@@ -496,11 +496,12 @@ def build_image_sent_module(task, d_inp, params):
     return pooler
 
 
-def build_single_sentence_module(task, d_inp, params):
+def build_single_sentence_module(task, d_inp, use_bert, params):
```

Comment:
Use keyword arguments where possible? (e.g. `use_bert: bool=False`)

Reply:
You mean when we actually call this? Or adding the typing annotations?

Reply:
Ideally, both?

Reply:
done (just for this function and calls to it though)